src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -661,6 +669,15 @@ private void DecodeWellKnownAttributeAppliedToMethod(ref DecodeWellKnownAttribut
             }
         }
 
+        internal ThreeState IsRuntimeAsyncEnabledInMethod
+        {
+            get
+            {
+                var data = GetDecodedWellKnownAttributeData();
```

Comment:
nit: consider inlining like other properties below
nit: for other similar properties, it looks like we use the same name as the underlying attribute data property. Consider using matching names too
```
internal ThreeState RuntimeAsyncMethodGenerationSetting => 
    GetDecodedWellKnownAttributeData()?.RuntimeAsyncMethodGenerationSetting ?? ThreeState.Unknown;
```